config/betterquesting/DefaultQuests.json

```diff
@@ -1940,15 +2097,15 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "Your first mandatory §mbig§r Project! This one is for Steel!\n\nThe Primitive Blast Furnace (PBF) is an inherently §5B A L A N C E D§r multiblock. You will need 132 clay, split into 66 clay dusts and 66 brick dusts.\nAs a pro tip: if you are grinding down using Mortars, do it with clay blocks and brick blocks.\n\nBuild the structure using JEI preview.\n\nAs for the Steel recipe, you will be rewarded if you made the Coke Oven earlier, as each Steel Ingot only requires §61 Coke§r instead of §62 Coal§r. Otherwise, you can simply use §6Charcoal§r as it is mostly free. Also, using Wrought Iron cuts the recipe duration by §a60%§r.\n\n\nSteel will be §9extremely important throughout the entire game§r, make a lot of it!\nSteel also has §6double the durability§r compared to regular Iron for GregTech tools. It also provides Diamond mining level.",
+          "desc:8": "Your first mandatory §mbig§r Project! This one is for Steel!\n\nThe Primitive Blast Furnace (PBF) is an inherently §5B A L A N C E D§r multiblock. If you do not have a §dMacerator§r yet, this multiblock will be §lfour times as expensive§r.\nYou will need 132 Clay, split into 66 Clay Dusts and 66 Brick Dusts.\nAs a pro tip: if you are grinding down using Mortars, do it with clay blocks and brick blocks.\n\nBuild the structure using JEI preview.\n\nAs for the Steel recipe, you will be rewarded if you made the Coke Oven earlier, as each Steel Ingot only requires §6one Coke§r instead of §6one Coal§r. Otherwise, you can simply use §6Charcoal§r as it is mostly free.\nUsing Wrought Iron cuts the recipe duration significantly... if you have the sanity to smelt thousands of nuggets beforehand.\n\n\nSteel will be §9extremely important throughout the entire game§r, make a lot of it!\nSteel also has §6double the durability§r compared to regular Iron for GregTech tools. It also provides Diamond mining level.",
```

Comment:
The brick and clay block tip may no longer apply with the latest version. You also say that "each Steel Ingot requires one Coke instead of one Coal." Doesn't this mean coal and coke are equivalent?